package.json

```diff
@@ -1,24 +1,50 @@
 {
-  "babel": {
-    "presets": [
-      "es2015"
-    ]
-  },
   "dependencies": {
+    "class-set": "^0.0.4",
+    "codemirror": "^5.39.2",
     "is-valid-email": "0.0.2",
+    "react": "^16.4.2",
+    "react-codemirror2": "^5.1.0",
+    "react-dom": "^16.4.2",
+    "react-router-dom": "^4.3.1",
     "url": "^0.11.0",
-    "valid-url": "^1.0.9",
-    "virtual-dom": "^2.1.1"
+    "valid-url": "^1.0.9"
+  },
+  "browserify": {
```

Comment:
I don't remember when we introduced browserify, but is this the right choice? Should we be using webpack?